warrior/WarriorCore/Classes/argument_datatype_class.py

```diff
@@ -64,27 +65,32 @@ def convert_arg_to_datatype(self):
             self.datatype = tuple
         elif self.arg_name.startswith('dict_'):
             self.datatype = dict
+        elif self.arg_name.startswith('file_'):
+            self.datatype = file
+            try:
+                self.arg_value = file(self.arg_value)
```

Comment:
Input conversion should to be relative to the testcase location not to "warriorframework/warrior".